lib/features/preview-support/PreviewSupport.js

```diff
@@ -94,24 +99,27 @@ PreviewSupport.prototype.getGfx = function(element) {
  * @param {Element} element The element to be moved.
  * @param {SVGElement} group The SVG group to add the preview to.
  * @param {SVGElement} [gfx] The optional graphical element of the element.
+ * @param {string} [className="djs-dragger"] The optional class name to add to the preview.
  *
  * @return {SVGElement} The preview.
  */
-PreviewSupport.prototype.addDragger = function(element, group, gfx) {
+PreviewSupport.prototype.addDragger = function(element, group, gfx, className = 'djs-dragger') {
   gfx = gfx || this.getGfx(element);
 
   var dragger = svgClone(gfx);
   var bbox = gfx.getBoundingClientRect();
 
-  this._cloneMarkers(getVisual(dragger));
+  this._cloneMarkers(getVisual(dragger), className);
 
-  svgAttr(dragger, this._styles.cls('djs-dragger', [], {
+  svgAttr(dragger, this._styles.cls(className, [], {
     x: bbox.top,
     y: bbox.left
   }));
 
   svgAppend(group, dragger);
 
+  svgAttr(dragger, 'data-complex-preview-element-id', element.id);

```

Comment:
What about namespacing it to the actual component, i.e. `data-preview-support-element-id`?

Reply:
Sure! 👍🏻

Reply:
Actually, this was a mistake. Makes no sense to call it `data-complex-preview`. 🤦🏻‍♂️